runtime/testnet/src/config/assets.rs

```diff
@@ -30,6 +30,7 @@ parameter_types! {
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
 	pub const NftsCollectionDeposit: Balance = 10 * UNIT;
+	pub const NftsCollectionApprovalDeposit: Balance = UNIT / 100;
```

Comment:
As above, I think the deposit function might be preferred for approvals, especially as we are storing the amount with the approval.